java/java/JavaParser.g4

```diff
@@ -599,46 +599,59 @@ methodCall
 expression
     // Expression order in accordance with https://introcs.cs.princeton.edu/java/11precedence/
     // Level 16, Primary, array and member access
-    : primary
-    | expression '[' expression ']'
+    : primary                                                       #ExpressionPrimary
```

Comment:
It's better to reverse naming here, name of the label should end with type, e.g. `PrimaryExpression`, `MethodCallExpression` and for binary operations label should not be plural, because it's only 1 operator in one node, so better to rename to `BinaryOperatorExpression`

Reply:
done